snuba/migrations/operations.py

```diff
@@ -101,3 +101,113 @@ def format_sql(self) -> str:
         columns = ", ".join([col.for_schema() for col in self.__columns])
 
         return f"CREATE MATERIALIZED VIEW IF NOT EXISTS {self.__view_name} TO {self.__destination_table_name} ({columns}) AS {self.__query};"
+
+
+class AddColumn(SqlOperation):
+    """
+    Adds a column to a table.
+    """
+
+    def __init__(
+        self,
+        storage_set: StorageSetKey,
+        table_name: str,
+        column: Column,
+        after: Optional[str],
+    ):
+        super().__init__(storage_set)
+        self.__table_name = table_name
+        self.__column = column
+        self.__after = after
+
+    def format_sql(self) -> str:
+        column = self.__column.for_schema()
+        optional_after_clause = (
+            f" AFTER {self.__after}" if self.__after is not None else ""
```

Comment:
This should deal with the case where a user passes an empty string as `after` (failing).